scripts/lib/aws/sagemaker.sh

```diff
@@ -0,0 +1,41 @@
+
+#!/usr/bin/env bash
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="$THIS_DIR/../../.."
+SCRIPTS_DIR="$ROOT_DIR/scripts"
+
+. $SCRIPTS_DIR/lib/common.sh
+. $SCRIPTS_DIR/lib/testutil.sh
+. $SCRIPTS_DIR/lib/aws.sh
+
+service_name="sagemaker"
+
+#################################################
+# functions for tests
+#################################################
+
+# print_k8s_ack_controller_pod_logs prints kubernetes ack controller pod logs
+# this function depends upon testutil.sh
+print_k8s_ack_controller_pod_logs() {
+  local ack_ctrl_pod_id=$( controller_pod_id )
+  kubectl logs -n ack-system "$ack_ctrl_pod_id"
+}
+
+setup_sagemaker_common_test_inputs() {
+  # uses non local variable for later use in tests
+  sagemaker_data_bucket="REPLACE_ME"
+  sagemaker_execution_role_arn="REPLACE_ME"
+}
+
+setup_sagemaker_common_test_inputs
+
+assert_aws_sagemaker_model_created() {
```

Comment:
Add a docstring for uniformity across the repo - [example](https://github.com/aws/aws-controllers-k8s/blob/5f6c642afbc0eaa292aa6e6e3e830bc14aa5345e/scripts/lib/aws/elasticache.sh#L28)